agir/front/components/app/routes.config.js

```diff
@@ -28,6 +28,8 @@ const GroupMap = lazy(() => import("@agir/carte/page__groupMap/GroupMap"));
 
 const ActivityPage = lazy(() => import("@agir/activity/ActivityPage"));
 
+const ToolsPage = lazy(() => import("@agir/front/toolsPage/toolsPage"));
```

Comment:
Les fichier qui contiennent des composants React sont nommé avec une majuscule par convention, comme les composants :
```suggestion
const ToolsPage = lazy(() => import("@agir/front/toolsPage/ToolsPage"));
```